recipes/cppkafka/all/conanfile.py

```diff
@@ -41,8 +41,8 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.81.0")
-        self.requires("librdkafka/2.0.2")
+        self.requires("boost/1.83.0", transitive_headers=True)
```

Comment:
Checked, this is necessary, no need to comment on this one this time, I'd rather save the recompilation, just writing here to confirm for future maintenance :)